Bugsnag/Client/BugsnagClient+AppHangs.m

```diff
@@ -62,4 +73,34 @@ - (void)appHangEnded {
     self.appHangEvent = nil;
 }
 
+- (BOOL)lastRunEndedWithAppHang {
```

Comment:
Does this need to be idempotent in case it gets called more than once? (short-circuiting if `self.appHangEvent` != nil)

Reply:
Not strictly, since it shouldn't be called more than once (like `computeDidCrashLastLaunch`)

Reply:
This has now been made idempotent 👍